test/src/main/java/org/openjdk/skara/test/TestHostedRepository.java

```diff
@@ -55,6 +55,30 @@ public TestHostedRepository(TestHost host, String projectName, Repository localR
         commitComments = new HashMap<>();
     }
 
+    /**
+     * Creates an instance without a backing local repository that will not support any actual repository interaction
+     */
+    public TestHostedRepository(String projectName) {
+        super(null, projectName);
+        this.host = null;
+        this.projectName = projectName;
+        this.localRepository = null;
+        pullRequestPattern = null;
+        commitComments = new HashMap<>();
+    }
+
+    /**
+     * Creates an instance without a backing local repository that will not support any actual repository interaction
+     */
+    public TestHostedRepository(TestHost host, String projectName) {
```

Comment:
Still need this constructor because in a test, we need a `TestHostedRepository` with `TestHost`